subprojects/build-cache-http/src/integTest/groovy/org/gradle/caching/http/internal/HttpBuildCacheServiceTest.groovy

```diff
@@ -303,7 +303,8 @@ class HttpBuildCacheServiceTest extends Specification {
         server.expect("/cache/${key.hashCode}", false, [method], new HttpServer.ActionSupport("return ${httpCode} broken") {
             @Override
             void handle(HttpServletRequest request, HttpServletResponse response) {
-                response.sendError(httpCode, "broken")
+                //noinspection GrDeprecatedAPIUsage
+                response.setStatus(httpCode, "broken")
```

Comment:
Is there a reason to insist on using `broken` as the message string? Why not switch to `setStatus(int)` instead, and lose the deprecation along with the custom message?

Reply:
Just because sending an error has slightly different semantics and the tests do not expect the right message depending on the error code. e.g 302 shoudn't be an error.